packages/esm-patient-chart-app/translations/en.json

```diff
@@ -45,11 +49,16 @@
   "loadVisitInfo": "Load Visit Info",
   "location": "Location",
   "male": "Male",
+  "markAlive": "Mark Alive",
+  "markAsAlive": "Are you sure, you want to mark patient as alive?",
```

Comment:
This same key was having value `Mark as Alive`. Please make sure every key has different values.

When extract-translations runs, it gives warnings when it finds same key having different values.